### PR TITLE
Created reference with explanation of pricing calculator

### PR DESCRIPTION
**Pull request by matthijskrul**
## Summary of the changes / Why this is an improvement
Implements [story](https://trello.com/c/gPgWvlbk/407-2-pricing-calculator-guide) to create a guide for the CrateDB Cloud pricing calculator, based on the [updated version](https://cr8dev.0a.at/products/cratedb-cloud-on-azure/) of the latter.

Note: this should not be merged until implementation of the pricing calculator update.

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/price-calculator.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+================
+
+The `price calculator`_ for CrateDB Cloud is a tool allowing you to estimate
+the price of a suitable Cloud subscription plan for your particular use case.
+The output of the price calculator depends on three things: the subscription
+service (Cloud provider), the subscription plan base configuration, and finally
+the specific data requirements for your Cloud service.
+
+.. NOTE::
+    Currently, the only available base configuration for fully supported
```
Comment (mechanomi) [inline docs/price-calculator.rst:14]:
```suggestion
.. NOTE::

    Currently, the only available base configuration for fully supported
```

File: docs/price-calculator.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The specs configuration is where the price calculator is most effective. It
+allows you to specify your expected (estimated) requirements for the Cloud
+service in terms of detailed specifications. The price calculator takes five
+inputs for these specifications:
+
+* The expected number of **ingests** per second. This means the number of
+  sensor or machine data readings expected to be generated into the database
+  per second.
+* The expected number **read queries** per second. This represents a
```
Comment (mechanomi) [inline docs/price-calculator.rst:40]:
```suggestion
  per second.
  
* The expected number **read queries** per second. This represents a
```

File: docs/price-calculator.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  multiplication of the number of data points to be queried, their update
+  frequency (per second), and the number of representations of each data point.
+* The estimated average payload size, in kilobytes (kb).
```
Comment (mechanomi) [inline docs/price-calculator.rst:43]:
```suggestion
  frequency (per second), and the number of representations of each data point.
  
* The estimated average payload size, in kilobytes (kb).
```

File: docs/price-calculator.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* The desired data retention policy, in days. This is the number of days a
+  given datum or record is to be retained in the database.
```
Comment (mechanomi) [inline docs/price-calculator.rst:45]:
```suggestion

* The desired data retention policy, in days. This is the number of days a
  given datum or record is to be retained in the database.
```

File: docs/price-calculator.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+* The number of replicas. The offered range varies between zero and two. More
```
Comment (mechanomi) [inline docs/price-calculator.rst:46]:
```suggestion

* The number of replicas. The offered range varies between zero and two. More
```

File: docs/price-calculator.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  replicas means more copies of the data stored across the nodes, thereby
+  providing additional security against failure but also additional storage
+  requirements.
+
+.. NOTE::
+    Short descriptions of each of these inputs are also found as tooltips on
```
Comment (mechanomi) [inline docs/price-calculator.rst:52]:
```suggestion
.. NOTE::

    Short descriptions of each of these inputs are also found as tooltips on
```

File: docs/price-calculator.rst [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+hourly rate basis.
+
+
+.. _price-calculator-provider:
+
+The Cloud provider
+==================
+
+The Cloud provider specifies the provider of the Cloud subscription. Currently
+we only support Azure subscriptions. It is intended to expand this to Amazon
```
Comment (mechanomi) [inline docs/price-calculator.rst:69]:
```suggestion
we only support Azure subscriptions. It is intended to expand this to Amazon
```

can you switch this to active voice? i.e., "We intend"

**Comment by matthijskrul** [comment]
I think this should cover the explanation section

**Pull request merged**